pkg/asset/installconfig/clustername.go

```diff
@@ -35,7 +35,17 @@ func (a *clusterName) Generate(parents asset.Parents) error {
 		validator = survey.ComposeValidators(validator, func(ans interface{}) error {
 			return validate.ClusterName1035(ans.(string))
 		})
+		if platform.GCP != nil {
+			validator = survey.ComposeValidators(validator, func(ans interface{}) error {
```

Comment:
This is going to replace the RFC-1035 validator above rather than adding to it.

Reply:
I changed the logic for the GCPClusterName to just validate against the goog and google parts and return nil if success. Do I set it to the old logic?

Reply:
We need the new check for goog and google as well as the old RFC-1035 check.

Reply:
Oh, sorry. I misread that the code was doing composition. This looks good as is.

Reply:
Yes but doesn't this chain the two validators together into one?